lib/salus/scanners/bandit.rb

```diff
@@ -0,0 +1,42 @@
+require 'bundler'
+require 'salus/scanners/base'
+
+# Bandit scanner to check for Python vulns.
+# https://github.com/presidentbeef/brakeman
```

Comment:
this link is not correct right?

Reply:
@ghbren Can you update this url to point at bandit?

Reply:
fixed